Java/PlatinumGenomes-variant-transformation/src/main/java/com/google/cloud/genomics/examples/TransformNonVariantSegmentData.java

```diff
@@ -153,8 +164,13 @@ private static TableSchema getTableSchema() {
     public void processElement(ProcessContext c) {
       Variant v = c.element();
 
+      HashMultiset genotypeCount = HashMultiset.create();  // This will typically hold counts for genotypes -1,0,1,2.
```

Comment:
Conceptually it is actually a set with convenience methods for getting the counts. It will also, yes, typically be implemented efficiently storing the counts and not multiple copies of duplicated elements. But it is still a set.

But I'll agree it is a point of view and not especially important here.